src/index.ts

```diff
@@ -90,6 +118,14 @@ class ServerlessPlugin {
     // Remove temp build folder
     fs.removeSync(path.join(this.originalServicePath, buildFolder))
   }
+
+  async cleanup(): Promise<void> {
```

Comment:
I think we can, since the after function actually uses originalServicePath, I think I can run the cleanup steps after everything in that function.